### ISSUE TITLE
Job timeout error message does not reflect maxPollDuration

### ISSUE THREAD

**Issue by donovanmuller** [keptn-sandbox/job-executor-service#49]
The job timeout message `max poll count reached for job %s. Timing out after 5 minutes` is confusing if `maxPollDuration` is set.

https://github.com/keptn-sandbox/job-executor-service/blob/c45ce5924e559f26b422e3c8a90c37f8587ca723/pkg/k8sutils/job.go#L186

The time value in the message should reflect the actual timeout set with `maxPollDuration`.

**Comment by didiladi** [comment]
@donovanmuller You are right, seems like we missed that. Thanks for pointing it out! :rocket: 

PS, we would be happy for any contribution! :)
